src/shared/ws2ipdef.rs

```diff
@@ -77,3 +77,10 @@ STRUCT!{struct SOCKADDR_IN6_LH {
   u: SOCKADDR_IN6_LH_u,
 }}
 pub type PSOCKADDR_IN6_LH = *mut SOCKADDR_IN6_LH;
+
+pub const IFF_UP: ULONG = 0x00000001;
+pub const IFF_BROADCAST: ULONG = 0x00000002;
+pub const IFF_LOOPBACK: ULONG = 0x00000004;
+pub const IFF_POINTTOPOINT: ULONG = 0x00000008;
+pub const IFF_POINTOPOINT: ULONG = IFF_POINTTOPOINT;
```

Comment:
As it turns out, I cannot actually find this definition in the Windows SDK. Where exactly did you find it?

This should likely be removed.